packages/suite/src/support/messages.ts

```diff
@@ -2353,7 +2353,7 @@ export default defineMessages({
         id: 'TR_DEVICE_SETTINGS_HOMESCREEN_EDITOR',
     },
     TR_DEVICE_SETTINGS_HOMESCREEN_IMAGE_SETTINGS_COLOR: {
-        defaultMessage: 'Use a JPG file ({width}×{height} px, up to {maxSizeKb} KB).',
+        defaultMessage: '{width}×{height} px, up to {maxSizeKb} KB',
```

Comment:
Check out also copy for T1/TS3.
<img width="882" height="142" alt="Screenshot 2025-09-24 at 18 17 51" src="https://github.com/user-attachments/assets/7cd9c060-e315-4e38-9db2-2e60b8182ccb" />
Perhaps it should use the same translation key and have one extra with the information about colour limitation.